dev/test-dependencies.sh

```diff
@@ -85,10 +85,15 @@ for HADOOP_PROFILE in "${HADOOP_PROFILES[@]}"; do
 
   echo "Generating dependency manifest for $HADOOP_PROFILE"
   mkdir -p dev/pr-deps
-  $MVN $HADOOP2_MODULE_PROFILES -P$HADOOP_PROFILE dependency:build-classpath -pl assembly \
-    | grep "Dependencies classpath:" -A 1 \
-    | tail -n 1 | tr ":" "\n" | rev | cut -d "/" -f 1 | rev | sort \
-    | grep -v spark > dev/pr-deps/spark-deps-$HADOOP_PROFILE
+  $MVN $HADOOP2_MODULE_PROFILES -P$HADOOP_PROFILE dependency:build-classpath -pl assembly | grep "Dependencies classpath:" -A 1 | tail -n 1 | tr ":" "\n" | awk -F '/' '{
+    artifact_id=$(NF-2);
+    version=$(NF-1);
+    jar_name=$(NF-0);
```

Comment:
$NF should be good enough here.

Reply:
Modified as suggested. Thanks for catching that.